### ISSUE TITLE
Change email from address for cancellations

### ISSUE THREAD

**Issue by jweiland** [jweiland-net/reserve#52]
The normal e-mails are now sent from info@pforzheim-web.de, but cancellations e-mails are still sent from webkoordination@pforzheim.de which causes SPF problems

**Comment by pascal20997** [comment]
Maybe the cancellation e-mails are a few days older? All mails are sent using `MailUtility::sendMailToCustomer()` so there should be no difference between the checkout and cancellation mails.

**Comment by jweiland** [comment]
Well, I just tested it today to make sure I have the current version.
But I will try just again now...

**Comment by jweiland** [comment]
Again a screenshot from a current cancellation:

![image](https://user-images.githubusercontent.com/3610278/83546214-9fee5080-a500-11ea-976c-25c9801dc961.png)

**Comment by pascal20997** [comment]
I tested it too on that website and it the checkout mail is sent from the wrong mail too. I checked the facility record and there is no custom mail or reply-to configured. Maybe someone removed it?

![image](https://user-images.githubusercontent.com/1337769/83603813-60158080-a575-11ea-82d9-56964b8aaba1.png)

![image](https://user-images.githubusercontent.com/1337769/83603605-03b26100-a575-11ea-806a-549ec2ab9fdc.png)
